RELEASING.md

```diff
@@ -0,0 +1,244 @@
+
+1. Merge `beta` to `production` (`git merge --log=876423 --no-ff beta`) and push.
+   Wait until [CircleCI](https://circleci.com/gh/metabrainz/musicbrainz-server) and
+   [Jenkins (Selenium)](https://ci.metabrainz.org/job/musicbrainz-server/) are happy
+   with this merge.
+
+2. Merge `production` to `master` and push.
+
+### Build Docker images
+
+Then, you must build new MusicBrainz Server Docker images from Jenkins, go to
+
+1. Go to https://ci.metabrainz.org/job/musicbrainz-docker-images/
+
+2. “Build with Parameters” (in the left), and enter `production` for _IMAGE_BRANCH_.
+
+A build can take some time (from a few minutes to half an hour).
+Once the image is built you can begin the deployment.
+
+### Announce the deployment
+
+Because any deployment may cause issues for MusicBrainz users and MetaBrainz staff,
+each deployment should be announced through the IRC channel and the website’s banner.
+
+[Set the banner message](https://musicbrainz.org/admin/banner/edit) to
+
+```html
+MusicBrainz servers are being updated, slowdowns may occur for a few minutes, thanks for your patience.
+```
+
+### Deploy to production
+
+See the private system administration wiki for instructions.
+
+### Update websites’ banner
+
+Then [unset the banner message on the main website](https://musicbrainz.org/admin/banner/edit)
+and [set the banner message on beta](https://beta.musicbrainz.org/admin/banner/edit) to
+
+```html
+Beta website is currently the same as the main website, nothing to be tested for now.
+```
+
+### Update Jira
+
+Now that you have done the release, you will need to update Jira:
+
+1. Make sure a new release version is present in the
+   [Jira project administration panel](https://tickets.metabrainz.org/projects/MBS?selectedItem=com.atlassian.jira.jira-projects-plugin:release-page).
+
+2. Close all tickets
+   [`status = "In Beta Testing" and project = MBS`](http://tickets.musicbrainz.org/secure/IssueNavigator.jspa?reset=true&jqlQuery=status+%3D+%22In+Beta+Testing%22+and+project+%3D+mbs)
+   as "Fixed."
+
+3. Make sure all the tickets have a type that makes sense, enough information
+   to be understandable, and the appropriate components set.
+
+4. Release the current version. All open tickets should be moved to the next
+   version.
+
+5. Archive any previous versions still marked as "Released", except for
+   the latest schema change release.
+
+### Blog
+
+[Create a new draft from the template](https://wordpress.com/post/blog.metabrainz.org?jetpack-copy=8634).
+The list of tickets can be copied from the auto-generated release notes,
+by going to the [Releases](http://tickets.musicbrainz.org/browse/MBS#selectedTab=com.atlassian.jira.plugin.system.project%3Aversions-panel) tab,
+selecting the version you just released,
+and clicking “Release Notes” in the top.
+
+Move sub-tasks of MBS-8609 to ending “React Conversion Task” section,
+and move other sub-tasks under “Other Task” (or just “Task” section if no React conversion task).
```

Comment:
```suggestion
Move sub-tasks of MBS-8609 to the “React Conversion Task” section at the end,
and move other sub-tasks under the “Other Task” section (rename to just “Task”
if there are no React conversion tasks in this release).
```

Reply:
(This paragraph was copied as-is.)

Committed as https://github.com/metabrainz/musicbrainz-server/pull/2340/commits/2b6a6ba93e309f1c3953db81a04eccd5ffd43f76